server/cmd/execute.go

```diff
@@ -37,9 +41,10 @@ func Execute(rootCmd *cobra.Command, envPrefix, defaultHome string) error {
 // CreateExecuteContext returns a base Context with server and client context
 // values initialized.
 func CreateExecuteContext(ctx context.Context) context.Context {
-	srvCtx := server.NewDefaultContext()
+	// srvCtx := server.NewDefaultContext()
 	ctx = context.WithValue(ctx, client.ClientContextKey, &client.Context{})
-	ctx = context.WithValue(ctx, server.ServerContextKey, srvCtx)
```

Comment:
See comment above, if we don't update the other helper this will break people not doing the migration (which I think we shouldn't enforce for people staying on baseapp)